src/EnergyPlus/WaterToAirHeatPumpSimple.cc

```diff
@@ -964,10 +965,13 @@ namespace WaterToAirHeatPumpSimple {
             state.dataWaterToAirHeatPumpSimple->MyOneTimeFlag = false;
         }
 
+        // substitute throughout
+        auto &simpleWatertoAirHP(state.dataWaterToAirHeatPumpSimple->SimpleWatertoAirHP(HPNum));
+
         if (state.dataWaterToAirHeatPumpSimple->MyPlantScanFlag(HPNum) && allocated(state.dataPlnt->PlantLoop)) {
             errFlag = false;
             ScanPlantLoopsForObject(state,
-                                    state.dataWaterToAirHeatPumpSimple->SimpleWatertoAirHP(HPNum).Name,
+                                    simpleWatertoAirHP.Name,
```

Comment:
There is opportunity for speed improvement with this sweeping change throughout specific functions. I did not want to do that here since it would change nearly every line of code.